ludwig/utils/tokenizers.py

```diff
@@ -996,9 +999,14 @@ def __init__(
                 super().__init__(pretrained_model_name_or_path, vocab_file)
 
             def _init_tokenizer(self, pretrained_model_name_or_path: str, vocab_file: str):
-                return torchtext.transforms.CLIPTokenizer(
-                    encoder_json_path=vocab_file, merges_path=pretrained_model_name_or_path
-                )
+                if torchtext_112:
+                    return torchtext.transforms.CLIPTokenizer(
+                        encoder_json_path=vocab_file, vocab_bpe_path=pretrained_model_name_or_path
+                    )
+                else:
+                    return torchtext.transforms.CLIPTokenizer(
```

Comment:
If I understand correctly, this param was changed in `0.12.1`?

Reply:
@tgaddair Afaik, torchtext only has 0.12 and 0.13 with no minor versions in between. So this changed in 0.13. What's interesting is that the docs show that these parameters are the same, but the actual underlying implementation has different parameters.

In either case, idk why the two tests are failing since I have torch 1.11 and torchtext 0.12 locally and the tests pass with this change (but were failing on my local before)

Reply:
Something to consider: they could go back and add a patch release for a security vulnerability, etc. So safest thing to do is to have a separate branches for `< 0.13` and `>= 0.13`.

Reply:
Looks like the failing test is for Python 3.7 only.

Reply:
Agreed, changed to branch on 0.13. It's weird that it's failing for Py37 since I have the exact same dependencies locally (torch 1.11 with torchtext 0.12) and it works